src/cli_plugin/install/install.js

```diff
@@ -59,11 +59,14 @@ export async function install(settings, logger) {
 
     del.sync(settings.tempArchiveFile, { force: true });
 
+    settings.plugins[0].pluginFolder = settings.override
```

Comment:
We should be using the same name everywhere, if the option is `folderName`, then this would also be `folderName` and `settings.folderName`.

Reply:
Made the naming consistent.